suseconnect-ng.spec

```diff
@@ -32,6 +32,26 @@ BuildRequires:  go >= 1.16
 Conflicts:      SUSEConnect
 Obsoletes:      SUSEConnect
 Provides:       SUSEConnect
+%if 0%{?fedora} || 0%{?rhel} || 0%{?centos_version}
```

Comment:
can SUSECoonnect be installed on these OSs?

Reply:
Yes, in theory. Not sure if anyone did use it, maybe for an in place migration to a SUSE OS. I copied this from the ruby version.